guides/source/action_controller_overview.md

```diff
@@ -730,6 +730,12 @@ class ChangesController < ApplicationController
 end
 ```
 
+Alternatively, you may specify the `around_action` inline, with the block yielding in the controller and action:
+
+```ruby
+around_action { |_controller, action| time(&action) }
+```
+
 Note that an "around" filter also wraps rendering. In particular, if in the example above, the view itself reads from the database (e.g. via a scope), it will do so within the transaction and thus present the data to preview.
```

Comment:
This sentence refers to the `wrap_in_transaction` example, and adding another code block in between the two makes it harder to follow.

It seems like the new example would fit better in the "Other Ways to Use Filters" section below, which begins with:

> While the most common way to use filters is by creating private methods and using *_action to add them, there are two other ways to do the same thing.

Reply:
Updated! Thank you

Reply:
cc / @eugeneius if you can take another look ✨